src/utils.c

```diff
@@ -795,11 +795,11 @@ gchar *utils_get_initials(const gchar *name)
  *  @param config A GKeyFile object.
  *  @param section The group name to look in for the key.
  *  @param key The key to find.
- *  @param default_value The default value which will be returned when @a section or @a key
- *         don't exist.
+ *  @param default_value The default value which will be returned when @a section
+ *         or @a key don't exist.
```

Comment:
No unrelated changes, please.

Reply:
Ok.  Will revert this when rebasing on the union code you will provide.